src/Microsoft.SqlTools.ServiceLayer/Connection/ReliableConnection/ReliableSqlConnection.cs

```diff
@@ -341,9 +373,13 @@ private SqlConnection OpenConnection()
                 if (_underlyingConnection.State != ConnectionState.Open)
                 {
                     _underlyingConnection.Open();
+
+                    // Asking if the server is sqldw or not
+                    IsSqlDwConnection(_underlyingConnection);
```

Comment:
Minor: why call it in here when you call it again in `SetDefaultAnsiSettings` just a few lines down? Is there a benefit / need to do so?